### ISSUE TITLE
Temp adjustment skips one change

### ISSUE THREAD

**Issue by MacPat** [luc-ass/homebridge-evohome#62]
Hi,

I've installe Homebridge and your plugin today on a RPI 3 and somehow one change is skipped if I change the temperature. All times are correct touch, and timezone is in sync.

[2020-10-21 0:51:05] [Evohome] Request to set target temperature to 15
[2020-10-21 0:51:09] [Evohome] The current time is 0:51:09
[2020-10-21 0:51:09] [Evohome] Schedule points for today (Wednesday)
[2020-10-21 0:51:09] [Evohome] - 07:30:00
[2020-10-21 0:51:09] [Evohome] - 16:30:00 -> next change
[2020-10-21 0:51:09] [Evohome] - 23:30:00
[2020-10-21 0:51:09] [Evohome] Setting target temperature for Woonkamer Thermostat to 15° until 16:30:00
[2020-10-21 0:51:11] [Evohome] Successfully changed temperature!

Do you have any idea how I can solve this?

**Comment by luc-ass** [comment]
I've tried to dig into this problem but got nowhere, as my system works fine. Does this problem still persist?

**Comment by luc-ass** [comment]
I just gave it another test run:
```
[1/17/2021, 11:18:52 AM] [Evohome] Current temperature of Wohnküche Thermostat is 21°
[1/17/2021, 11:18:53 AM] [Evohome] Request to set target temperature to 21
[1/17/2021, 11:18:58 AM] [Evohome] The current time is 11:18:58
[1/17/2021, 11:18:58 AM] [Evohome] Schedule points for today (Sunday)
[1/17/2021, 11:18:58 AM] [Evohome] - 06:20:00
[1/17/2021, 11:18:58 AM] [Evohome] - 16:00:00 -> next change
[1/17/2021, 11:18:58 AM] [Evohome] - 23:30:00
[1/17/2021, 11:18:58 AM] [Evohome] Setting target temperature for Wohnküche Thermostat to 21° until 16:00:00
[1/17/2021, 11:18:58 AM] [Evohome] Successfully changed temperature!
```
Seems to work fine in my case. Does this happen throughout the day, or is it limited to specific times? I was wondering wether this only happens between 23:59:59 and 0:59:59 as the hour before 01:00:00 is missing the leading "double zero".

**Comment by PuzzledUser** [comment]
A related issue but with perhaps a clear cause (current time reported wrong) https://github.com/luc-ass/homebridge-evohome/issues/100

**Comment by luc-ass** [comment]
This issue is probably solved by the proposed solution in #100
